### PR TITLE
#93 add become a speaker form

### LINKED ISSUE #93: Become a speaker
- At speakers#index page add button "become a speaker"
- Form: name, email, text (if logged in name and email is unnecessary)
- Send mail to admin with form data

### PR DESCRIPTION
**Pull request by oookli**
Fixed #93

thanks @denysmedynskyi for help

### CODE REVIEW

File: app/mailers/notify_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class NotifyMailer < ApplicationMailer
+  def become_speaker(params)
+    mail(from: "#{params[:name]} <#{params[:email]}>", subject: "I want be a speaker", to: PIVORAK_EMAIL) do |format|
```
Comment (sveredyuk) [inline app/mailers/notify_mailer.rb]: - destruct params to local variables please
- "I want to be a speaker"

File: app/views/members/_become_speaker_form.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+= form_tag become_speaker_path do
+  = text_field_tag :name, current_user&.first_name,
+                          required:    true
+  br
+  br
```
Comment (sveredyuk) [inline app/views/members/_become_speaker_form.slim:5]: should we allowed `br` make format ?

File: config/routes.rb [hunk 3]
```diff
@@ -19,6 +19,7 @@
   resource  :chat,    only: %i[show create],      controller: :chat
   resource  :profile, only: %i[show edit update], controller: :profile
   resources :members, only: %i[index show]
+  resource :become_speaker, only: %i[create], controller: :become_speaker
```
Comment (sveredyuk) [inline config/routes.rb]: i think this should be embedded into members

**Comment by oookli** [comment]
@sveredyuk thanks, I'll fix all

File: app/views/members/_become_speaker_form.slim (hunk 2 above)
Comment (oookli) [reply app/views/members/_become_speaker_form.slim:5]: I think, we should wait for a design first, yes `br` is not good for formatting form or other things, but in this case is not so bad.

**Review by sveredyuk — APPROVED** [review]
@oookli good job. Thx for help!

**Pull request merged**